code/modules/mob/living/carbon/carbon.dm

```diff
@@ -347,6 +347,7 @@
 				W.layer = initial(W.layer)
 				W.plane = initial(W.plane)
 		changeNext_move(0)
+		update_equipment_speed_mods() // In case cuffs ever change speed

```

Comment:
You may as well unindent this. This proc needs to be run whenever equipment changes anyway, even handcuffs

Reply:
Good point, done.